eva-server/src/main/java/uk/ac/ebi/eva/server/ws/ga4gh/beaconv2/BeaconServiceV2.java

```diff
@@ -161,10 +166,13 @@ private void checkParameters(String chromosome, String referenceBases, Long star
             throw new IllegalArgumentException("Please provide reference bases");
         }
 
-        if (start != null && start < 0) {
+        if (start == null) {
```

Comment:
If a `start` is provided we need to check there is either an `end` or a `reference`

If a `start` is not provided (imprecise positions) then we need to check all fields `startMin` + `startMax` + `endMin` + `endMax` are provided

There are some missing checks here

Reply:
Actually `referenceBases` is a parameter that is always required and check for that is done just above this, so I don;t think that it is needed to be checked again.
I will add the check for endMin and endMax here :)

Reply:
Added :)